### DISCUSSION TITLE
Feature Request: Support for Input Booleans

### DISCUSSION THREAD

**Discussion by Nuuki9** [jseidl/magic-areas#450] [Ideas / Feature Requests]
It would be really useful to be able to add input booleans as a trigger for MA. As Automations can't directly update sensors, helpers are a common building block for creating custom triggers etc, and being able to add these would be really useful.

**Comment by jseidl** [comment]
As a trigger for presence? You can create a `binary_sensor` of device class `presence` template sensor based-off that input boolean. Magic Areas can't automatically consider `input_booleans` because they will not be always related to presence.
